microsoft-edge/devtools/network/reference.md

```diff
@@ -734,97 +1390,220 @@ To see both the transferred size and the uncompressed size of a resource at the
+<!-- older para: -->
+You can't filter requests, and you can't save a single request.
+
+
+To save all network requests to a HAR file, including sensitive data:
+
+1. In DevTools, select **Customized and control DevTools** > **Settings**.
+
+   The **Settings** > **Preferences** page opens.
+
+1. Scroll down to the **Network** section.
+
+1. Select the checkbox **Allow to generate HAR with sensitive data**.
+
+1. Click the **Close** (![Close Settings button](./reference-images/close-settings-button.png)) button.
+
+1. In the **Network** tool, right-click any request in the **Requests** table, and then select **Copy** > **Copy all as HAR (with sensitive data)**.
+
+   Or, in the action bar at the top, click the **Export HAR (either sanitized or with sensitive data)** (![Export HAR icon](./reference-images/export-har-sanitized-icon.png)) button, and then select **Export HAR (with sensitive data)**:
+
+   ![Selecting 'Copy all as HAR (with sensitive data)'](./reference-images/export-har-button.png)
 
 
 <!-- ------------------------------ -->
-#### Copy one or more requests to the clipboard
+#### Import a HAR file into DevTools for analysis
+<!-- bottom of https://developer.chrome.com/docs/devtools/network/reference/#save-as-har -->
+
+After you have a HAR file, you can import it back into DevTools for analysis by using [HAR Analyzer](https://toolbox.googleapps.com/apps/har_analyzer/).
+
+To import a HAR file into DevTools for analysis:
+
+* In the action bar at the top of the **Network** tool, click the **Import HAR file** (![Import HAR file icon](./reference-images/import-har-file-icon.png)) button.
+
+* Or, drag the HAR file onto the **Requests** table.
+
+The **Network** tool reads and shows initiators for the requests that are imported from the HAR file.  For initiators, see [Display a log of requests](#display-a-log-of-requests), above.
+
+
+<!-- ------------------------------ -->
+#### Copy network requests to the clipboard
+<!-- Copy a request, a filtered set of requests, or all of them to the clipboard  https://developer.chrome.com/docs/devtools/network/reference/#copy -->
+
+Under the **Name** column of the **Requests** table, right-click a request, hover over **Copy**, and then select one of the following options.
+
+To copy a single request, its response, or <!-- todo: its? -->stack trace:
+
+| Name | Details |
+| --- | --- |
+| **Copy URL** | Copy the URL of the request to the clipboard. |
+| **Copy as cURL (cmd)** | Copy the request as a cURL command. |
+| **Copy as cURL (bash)** |  |
+| **Copy as PowerShell** | Copy the request as a PowerShell command. |
+| **Copy as fetch** | Copy the request as a fetch call. |
+| **Copy as fetch (Node.js)** | Copy the request as a Node.js fetch call. |
+| **Copy response** | Copy the response body to the clipboard. |
+| **Copy stack trace** | Copy the request's stack track to the clipboard. |
```

Comment:
```suggestion
| **Copy stack trace** | Copy the request's stack trace to the clipboard. |
```
@captainbrosset - tech review this change. 
upstream
https://developer.chrome.com/docs/devtools/network/reference/#copy
says "stack track", typo?

Reply:
changed upstream text from "stack track" to "stack trace" 
@captainbrosset